.azure-pipelines/default.yml

```diff
@@ -0,0 +1,40 @@
+trigger:
+- master
+
+jobs:
+- job: 'Windows'
+
+  pool:
+    vmImage: 'vs2017-win2016'  
+
+  steps:
+  - script: |
+     python -m venv .\
+     
+     pip install git+https://github.com/lmazuel/swagger-to-sdk
+     pip install -e "git+https://github.com/Azure/azure-sdk-for-python#subdirectory=azure-sdk-tools&egg=azure-sdk-tools"
+
+    displayName: 'Prep Environment, Install Packages copy'
+
+  - powershell: |
+     $inputFilterSet="$(build_targeting_string)"
+     
+     if ($inputFilterSet -eq "")
+     {
+         $inputFilterSet = "azure-*"
```

Comment:
Is there a reason you want to default the value here as opposed to in the build definition? Also is `azure-*` the correct filter for the client libraries?

Reply:
There is no defaulting in the build def, I prefer to error case to success, but that's something we can decide on as a team.